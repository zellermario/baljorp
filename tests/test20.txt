create SJ m 0
create Eszkimo e m
create Sator s field m
digobject e
useobject s e
useobject s e
