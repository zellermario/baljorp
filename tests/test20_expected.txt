Fields:
SJ m (0) - object: x
	neighbours: []

Buildings:
FelepitettSator s on m

Actors:
> Eszkimo e (heat: 5) on m
  Inventory: []